src/Sha1Test.java

```diff
@@ -92,7 +92,18 @@ private static void realMain(String uri, String path) {
 			} catch(FileNotFoundException e) {
 				System.err.println("Not found, let's ignore that mirror.");
 			} catch(SSLException ssle) {
-				System.err.println("An SSL exception has occured:" + ssle.getMessage());
+				System.err.println("An SSL exception has occurred: " + ssle.getMessage());
+				Throwable cause = ssle.getCause();
+				if (cause != null) {
+					while (cause.getCause() != null)
+						cause = cause.getCause();
+					System.err.println("Cause: " + cause.getMessage());
+					if (cause.getMessage() == "Prime size must be multiple of 64, and can only range from 512 to 1024 (inclusive)") {
```

Comment:
Is this message not localized or will it be different if the JVM isn't set to English?